server/src/main.rs

```diff
@@ -37,7 +37,7 @@ async fn main() -> std::io::Result<()> {
             .service(update_key)
             .service(update)
     })
-    .bind("127.0.0.1:8080")?
+    .bind(("0.0.0.0", 8080))?
```

Comment:
Is this running on the same port or different? as the previous version. We want both

Reply:
had a confusion while thinking through , will raise a fix